pkg/config/containers.conf

```diff
@@ -497,3 +497,9 @@ default_sysctls = [
 # TOML does not provide a way to end a table other than a further table being
 # defined, so every key hereafter will be part of [volume_plugins] and not the
 # main config.
+
+[secret]
+# driver="pass"
```

Comment:
This should be "file" to indicated the default.

Reply:
changed.